projeto_semear/Pose initial_pose
projeto_semear/Pose goal_pose
---
uint8[] path
